script/hassfest/requirements.py

```diff
@@ -74,8 +81,42 @@ def validate(integrations: dict[str, Integration], config: Config):
         validate_requirements(integration)
 
 
+def validate_requirements_format(integration: Integration) -> bool:
+    """Validate requirements format.
+
+    Returns if an error was added.
+    """
+    start_errors = len(integration.errors)
+
+    for req in integration.requirements:
+        if " " in req:
+            integration.add_error(
+                "requirements",
+                f'Requirement "{req}" contains a space',
+            )
+            continue
+
+        pkg, sep, version = req.partition("==")
+
+        if not sep:
+            integration.add_error(
+                "requirements",
+                'Requirements need to be pinned "<pkg name>==<version>".',
```

Comment:
```suggestion
                f'Requirement {req} need to be pinned "<pkg name>==<version>".',
```
In case multiple requirements in the integration